minecraft-server/start-minecraft.sh

```diff
@@ -86,6 +89,31 @@ case $TYPE in
 
 esac
 
+# If supplied with a URL for a world, download it and unpack
+case "X$WORLD" in
+  X[Hh][Tt][Tt][Pp]*[Zz][iI][pP])
+    echo "Downloading world via HTTP"
+    echo "$WORLD"
+    wget -q -O - "$WORLD" > /data/world.zip
```

Comment:
Just an FYI you can instead compact that to

    wget -q -O /data/world.zip "$WORLD"

hmm...or use the `-O -` and pipe into the unzip